pkg/apis/navigator/types.go

```diff
@@ -140,6 +137,9 @@ type ImageSpec struct {
 }
 
 type NavigatorClusterConfig struct {
+	Version semver.Version
+	Image   *ImageSpec
```

Comment:
(as in other comment) I don't think we can be sure that all applications will use semver, and I'm also not sure if in future we'll need to support different ImageSpecs per node pool (or maybe multiple images per DB).

For now, it might be easiest to keep this where it is? WDYT?